pyproject.toml

```diff
@@ -15,11 +15,16 @@ description = "programatic access to conda-forge's metadata"
 dynamic = ["version"]
 dependencies = [
     "requests",
-    "pyyaml"
+    "ruamel.yaml"
 ]
 license = {file = "LICENSE"}
 readme = "README.md"
 
+[project.optional-dependencies]
+oci = [
+  "conda-oci-mirror"
```

Comment:
Is this on pypi or do we need to put a git url here?

Reply:
We can put a URL too. I'll use the tag URL.

Reply:
Let's see now.